data/sightings2-schema.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "http://json-schema.org/draft/2019-09/schema#",
+  "definitions": {
+    "sightingObject": {
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "examples": ["2.0"],
+          "const": "2.0"
+        },
+        "id": {
+          "type": "string",
+          "format": "uuid",
+          "examples": ["37146915-6b2e-485d-a436-6b8df66b8950"]
+        },
+        "start_time": {
+          "type": "string",
+          "format": "date-time",
+          "examples": ["2022-01-01T08:12:00Z"]
+        },
+        "detection_type": {
+          "type": "string",
+          "examples": ["human_validated"],
+          "enum": ["human_validated", "raw"]
+        },
+        "tid": {
+          "type": "array",
+          "items": {
```

Comment:
I noticed you made `tid` required but an empty array is considered a value. Suggest adding `"minItems": 1,`.